__tests__/Method-test.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {Method} from '../src/containers/Method';
+import { mountWithIntl, shallowWithIntl } from '../src/helpers/intl-enzyme-test-helper.js';
+import methodData from './method_data.json';
+
+function setup() {
+  const props = {
+    intl: {
+        formatDate: jest.fn(),
+        formatTime: jest.fn(),
+        formatRelative: jest.fn(),
+        formatNumber: jest.fn(),
+        formatPlural: jest.fn(),
+        formatMessage: jest.fn(),
+        formatHTMLMessage: jest.fn(),
+        now: jest.fn(),
+    },
+    params: {nodeID: 387}
+  }
+
+  const enzymeWrapper = shallowWithIntl(<Method {...props} />)
+  return {
+    props,
+    enzymeWrapper
+  }
+}
+
+describe('containers', () => {
+  describe('Method', () => {
+    it('should render proper data for method', () => {
+
+      const { enzymeWrapper } = setup()
+      enzymeWrapper.setState({data: methodData})
+      expect(enzymeWrapper.find('.sub-heading').length).toBe(2);
+      expect(enzymeWrapper.find('h2.case-title').text()).toBe('Petition');
```

Comment:
Same comment about the brittleness of this test.  I would say let's keep the data-content validity testing to the API test suite, and use the front-end test to focus on whether the API is returning well-formed content (but not specifically what content).